ivadomed/loader/utils.py

```diff
@@ -219,7 +220,7 @@ def get_subdatasets_subject_files_list(split_params, df, path_output, subject_se
         train_lst, valid_lst, test_lst = get_new_subject_file_split(df=df,
                                                                     split_method=split_params[SplitDatasetKW.SPLIT_METHOD],
                                                                     data_testing=split_params[SplitDatasetKW.DATA_TESTING],
-                                                                    random_seed=split_params[SplitDatasetKW.RANDOM_SEED],
+                                                                    random_seed=random_seed,
```

Comment:
I am worried here about retro-compatibility. Old config files where the seed is specified in the split dataset will probably crash. I think it's find to move it, but we should make sure at least for now that the current config files don't crash. There is this file that helps to deal with these scenarios: https://github.com/ivadomed/ivadomed/blob/master/ivadomed/config_manager.py#L70

Reply:
People using ivadomed use these config files as templates, but often have copies. For some studies, we have dozens of config file, so it's not viable to change the config file every time the code changes. When possible, we like to make everything retrocompatible which is why we created the class ConfigManager. This way it will support the new and previous versions of the config file.

Reply:
Well, noted.